specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/ResourceProvider.json

```diff
@@ -1051,6 +1067,10 @@
         "hostingEnvironment": {
           "description": "Name of App Service Environment where app or App Service plan should be created.",
           "type": "string"
+        },
+        "isXenon": {
+          "description": "<code>true</code> if App Service plan is running as a windows container",
+          "type": "boolean"
```

Comment:
above we are renaming xenonFxVersion to windowsFxVersion, which means having `isXenon` property instead of `isWindows` is inconsistent right?

Reply:
This distinguishes between windows and xenon windows and is therefore expected.